include/TPP/Dialect/Check/CheckDialect.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The IREE Authors
```

Comment:
I don't think it's necessary. We can say this in the commit message, in the RFC discussion, etc.

Let's keep those like the LLVM ones, because even if IREE wanted to upstream them, they'd have to drop their copyright comments.